src/containers/SignupContainer/fetchAuth.ts

```diff
@@ -0,0 +1,20 @@
+import fetch from '@/utils/request';
+import { IUserAuthResponseMessage } from '@/constants/types';
+import { FormValues } from '@/containers/SignupContainer/form';
+
+const FetchAuth = async (user: FormValues) => {
```

Comment:
rename to fetchAuth. Also, no need to create different files for different requests. So I'd suggest you change this to `fetchAPI` and then, use the `process.env.NEXT_PUBLIC_API` in the uri plus the route that you get as a parameter. And, use that everywhere.